src/test/java/org/isf/medicalsinventory/Tests.java

```diff
@@ -260,7 +262,8 @@ void testIoGetMedicalInventory() throws Exception {
 	void testMgrNewMedicalInventory() throws Exception {
 		Ward ward = testWard.setup(false);
 		MedicalInventory medicalInventory = testMedicalInventory.setup(ward, false);
-		MedicalInventory newMedicalInventory = medicalInventoryManager.newMedicalInventory(medicalInventory);
+		List<MedicalInventoryRow> medicalInventoryRows = new ArrayList<MedicalInventoryRow>();
```

Comment:
```suggestion
		List<MedicalInventoryRow> medicalInventoryRows = new ArrayList<>();
```
Explicit type replaced with `<>`